recipes/graylog-forwarder/files/graylog-forwarder.service

```
@@ -0,0 +1,26 @@
+[Unit]
+Description=Graylog Cloud Forwarder
```

Let's not use "Cloud" naming here. This might confuse people once this is also available for on-premise customers.
```suggestion
Description=Graylog Forwarder
```